### ISSUE TITLE
SRAL issues using NVDA

### ISSUE THREAD

**Issue by parker13435** [m1maker/SRAL#12]
When using NVDA with SRAL, it does some strange things when speaking text. Firstly, it does not work correctly with apostrophes. When SRAL speaks text that contains apostrophes, it puts extra spaces around them. Here is an example:

Original text:
"The code isn't working as it was expected to work. It's not displaying things properly sometimes, and that's a thing that shouldn't be happening."

After speaking the text through SRAL:
"The code isn  '  t working as it was expected to work. It  '  s not displaying the proper punctuation, and that  '  s a thing that shouldn  '  t be happening."

I think this is an issue with the controller client using SSML or Speech Synthesis Markup Language. Please look into this.

Another thing is that it seems like SRAL was configured so that when using the speech history addon for NVDA, none of the text that is spoken is added to the speech history buffer. I ran and analyzed the example code that came with the NVDA controller client, and it seems like the messages that use SSML are not added. I wonder if SRAL is using SSML messages and that is why these issues are happening. Please look into these issues when possible.

**Comment by m1maker** [comment]
Hello.

Thanks for your comments!
## First
I have already opened an issue in NVDA on the topic of placing spaces between apostrophes, SpeakSsml does this.
## Second
Yes, SRAL does use SSML for NVDA even for the regular Speak function to handle punctuation.
And about SpeechHistory, yes, indeed, it does not buffering SSML speech.
I'll take a look at it's code.

Thanks again for noticing this.
